packages/typespec-client-generator-core/test/package.test.ts

```diff
@@ -2953,7 +2960,9 @@ describe("typespec-client-generator-core: package", () => {
         sdkPackage.models.map((x) => x.name).sort(),
         ["CheckupCollectionWithNextLink", "Checkup", "PetStoreError", "CheckupUpdate"].sort()
       );
-      const createOrUpdate = sdkPackage.clients[0].methods[0];
+      const client = sdkPackage.clients[0].methods.find((x) => x.kind === "clientaccessor")
+        ?.response as SdkClientType<SdkHttpOperation>;
```

Comment:
Do we have test case for non-initializable subclients? Just curious what will `sdkPackage.clients` look like and how to initialize subclient.

Reply:
i think all current sub clients are non-initializable, they all have initialization with private access.

Reply:
Yeah, but TCGC should consider this case as well, we can add those cases when we met.

Reply:
non-initializable subclients are operation groups, which is what we currently support. we have a bunch of tests for this rn, so no need to add more